math/exprtk/Portfile

```diff
@@ -0,0 +1,25 @@
+PortSystem                  1.0
+PortGroup                   github 1.0
+
+github.setup                ArashPartow exprtk 0.0.3
+github.tarball_from         archive
+categories                  math devel
+maintainers                 @arashpartow
+platforms                   darwin
```

Comment:
this is the default snd should be removed; "port lint" has and the CI shows you a warning about it.